src/views/Nominate.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+    <div class="container">
+        <div class="nominate-container">
+            <div class="content_wrapper">
+                <h1 class="title_nominate">
+                    Nominate a <span class="highlight">developer</span>
+                </h1>
+                <p class="body_text">
+                    Help us build the directory of Developers thriving in the field of programming.
+                    If you know someone who deserves to be on the list, nominate them by clicking the Nominate button and filling the form.
+                    And hey, it's alright to nominate yourself😎
+                </p>
+                <a href="#" rel="noreferrer" class="nominate_button button_primary">Nominate</a>        
+            </div>
+        </div>           
+    </div>
+</template>
+
+<style lang="scss" scoped>
+    .container {
+        margin-top: 33px;
+    }
+    
+    .nominate-container {
+        display: flex;
+        flex: 1 1 auto;
+        flex-direction: column;
+        justify-content: flex-start;
+        align-items: center;
+        padding: 5rem;
+    }
+
+    .content_wrapper {
+        display: flex;
+        flex-direction: column;
+        max-width: 720px;
+        align-items: baseline;
+    }
+
+    .body_text {
+        font-size: 0.97rem;
+        font-family: Poppins,Source Sans Pro,sans-serif;
+        text-align: justify;
+        margin: 0;
+    }
+
+    .title_nominate {
+        font-size: 3.0rem;
+        margin: 5.0rem 0 2.0rem;
+        font-family: Poppins,Source Sans Pro,sans-serif;
+        font-weight: bold;
+        color: black;
+    }
+
+    .nominate_button {
+        text-align: center;
+        width: 200px;
+        display: inline-block;
+        height: 50px;
+        display: flex;
+        justify-content: center;
+        flex-direction: column;
+        text-decoration: none;
+        color: #fff;
+        background: #343a40;
+        border-radius: 6px;
+        font-family: "Poppins";
+        font-size: 1.3rem;
+        margin-top: 50px;
+    }
+
+    .button_primary {
```

Comment:
better create a .btn class with the basic style and later create other class with the variation, in this case `primary`
```css
.btn {
	color: black;
    background-color: white;
	...
	
	&.primary {
		background-color: blue;
	}

	&.danger {
		background-color: red;
	}
}
```

The usage will be something like
```html
<button class="btn">Cancel</button>
<button class="btn primary">Accept</button>
<button class="btn danger">Delete</button>
````